kubernetes/elastic-helm/configmap-elastic.yaml

```
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: elastic-otelcol-agent
+  namespace: default
+  labels:
+    app.kubernetes.io/name: otelcol
+
+data:
+  relay: |
+    connectors:
+      spanmetrics: {}
+    exporters:
+      debug: {}
+      otlp/elastic:
+        endpoint: ${ELASTIC_APM_ENDPOINT}
```

```suggestion
        endpoint: ${env:ELASTIC_APM_ENDPOINT}
```
Hmm, isn't the `env:` prefix required here? I'm always confused with that 😄  but further down we have  `${env:MY_POD_IP}`